From b7e6c56a78d272212dc63361c2602f2e6084b234 Mon Sep 17 00:00:00 2001
From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Fri, 28 Oct 2016 11:08:39 +0200
Subject: [PATCH 06/55] spice-vdagent.desktop: Set NoDisplay=true

Ubuntu's Unity still ships the Startup Applications app
(gnome-session-properties).

Setting NoDisplay=true means this autostart file won't show up in there
since this isn't something we really want users to unknowingly disable
that easily.

Resolves: https://bugs.freedesktop.org/show_bug.cgi?id=98465
---
 data/spice-vdagent.desktop | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/spice-vdagent.desktop b/data/spice-vdagent.desktop
index d152c12..3f91af4 100644
--- a/data/spice-vdagent.desktop
+++ b/data/spice-vdagent.desktop
@@ -6,3 +6,4 @@ Terminal=false
 Type=Application
 Categories=
 X-GNOME-Autostart-Phase=Initialization
+NoDisplay=true
-- 
2.16.1

